### ISSUE TITLE
(W10) Duplicate Search Icon

### ISSUE THREAD

**Issue by iitzRichie** [valinet/ExplorerPatcher#2788]
[As of the latest version of Explorer Patch, a second Search Icon (or Box) now appears on my Taskbar. I have my Taskbar aligned to the center, and the Search Icon is in the center, but also a second one exists on the left.](https://i.imgur.com/t2o5eV3)

Windows 10 Home v22H2
Explorer Patch v22621.2861.62.2

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by iitzRichie** [comment]
> Please post windows version and build. You can find this by running winver.
> 
> Please include what settings you changed in explorer patcher properties application. These should be screen shots.
> 
> Please include any other software you have installed that modifies the UI of windows.
> 
> Just a note, you can upload jpg files here. You don't have to upload it to imgur and the link to what you uploaded.

Updated OP. (Note: the only settings I have changed in explorer patcher properties application are Taskbar related settings, hence why there is only one image)

**Comment by pyrates999** [comment]
you got a preview update installed: https://support.microsoft.com/en-au/topic/january-23-2024-kb5034203-os-build-19045-3996-preview-d9540687-af96-46ba-9192-88fe44833561

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

In the EP properties application, in taskbar, I would set search to hidden to see if that fixes it.

**Comment by iitzRichie** [comment]
> Preview updates are also not fully tested by Microsoft and thus could have bugs.

> You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

I was installing them because at one point I was dealing with a pretty major issue in which people suggested that I download preview updates to fix it as well as always be sure to have the latest updates.

> In the EP properties application, in taskbar, I would set search to hidden to see if that fixes it.

Yeah, I did this, but it hides both search icons.

**Comment by iitzRichie** [comment]
> Try uninstalling the preview update and don't install preview updates as they are not needed.
> 
> Unless you like being a beta tester and don't mind reporting bugs with EP.

Yeah, I don't mind. Reporting it was only my intention here anyway.

**Comment by pyrates999** [comment]
If uninstalling the preview update fixed it, please indicate that.

**Comment by iitzRichie** [comment]
> If uninstalling the preview update fixed it, please indicate that.

I have not tried that yet. But I was replying to what you said about being a beta tester, and no I don't mind being a beta tester and reporting bugs with EP. My intention here was only to report this bug.

**Comment by iiTzRichie** [comment]
**Update**: As of 6/29/24 this issue still exists

___________________________________________________

_Note: I am no longer running any version of Windows Preview updates_

Windows 10 Home v22H2 (OS Build 19405.4529)
Explorer Patcher v22621.3527.65.5

**Comment by pyrates999** [comment]
> **Update**: As of 6/29/24 this issue still exists
> 
> _Note: I am no longer running any version of Windows Preview updates_
> 
> Windows 10 Home v22H2 (OS Build 19405.4529) Explorer Patcher v22621.3527.65.5

I think you meant 19045.4529.

**Comment by pyrates999** [comment]
What if you move the start button back to the left?

**Comment by iiTzRichie** [comment]
> I think you meant 19045.4529.

Yes. Apologies.

> What if you move the start button back to the left? Is search still shown twice?

Just tested it. No, search is not shown twice.

> Can you right click on the taskbar, go to search and set it to hidden and then back to show search icon?

Just tested it. Didn't fix anything.

**Comment by pyrates999** [comment]
Ok, please wait for a developer to respond.
